analysis/topeft_run2/run_analysis.py

```diff
@@ -32,85 +32,174 @@
+    )
+    parser.add_argument("--nworkers", "-n", default=8, help="Number of workers")
+    parser.add_argument(
+        "--chunksize", "-s", default=100000, help="Number of events per chunk"
+    )
+    parser.add_argument(
+        "--nchunks",
+        "-c",
+        default=None,
+        help="You can choose to run only a number of chunks",
+    )
+    parser.add_argument(
+        "--outname",
+        "-o",
+        default="plotsTopEFT",
+        help="Name of the output file with histograms",
+    )
+    parser.add_argument(
+        "--outpath", "-p", default="histos", help="Name of the output directory"
+    )
+    parser.add_argument(
+        "--treename", default="Events", help="Name of the tree inside the files"
+    )
+    parser.add_argument(
+        "--do-errors", action="store_true", help="Save the w**2 coefficients"
+    )
+    parser.add_argument(
+        "--do-systs", action="store_true", help="Compute systematic variations"
+    )
+    parser.add_argument(
+        "--split-lep-flavor",
+        action="store_true",
+        help="Split up categories by lepton flavor",
+    )
+    parser.add_argument(
+        "--offZ-split", action="store_true", help="Split up 3l offZ categories"
+    )
+    parser.add_argument(
+        "--tau_h_analysis", action="store_true", help="Add tau channels"
+    )
+    parser.add_argument("--fwd-analysis", action="store_true", help="Add fwd channels")
+    parser.add_argument(
+        "--skip-sr", action="store_true", help="Skip all signal region categories"
+    )
+    parser.add_argument(
+        "--skip-cr", action="store_true", help="Skip all control region categories"
+    )
+    parser.add_argument(
+        "--do-np",
+        action="store_true",
+        help="Perform nonprompt estimation on the output hist, and save a new hist with the np contribution included. Note that signal, background and data samples should all be processed together in order for this option to make sense.",
+    )
+    parser.add_argument(
+        "--do-renormfact-envelope",
+        action="store_true",
+        help="Perform renorm/fact envelope calculation on the output hist (saves the modified with the the same name as the original.",
+    )
+    parser.add_argument(
+        "--wc-list",
+        action="extend",
+        nargs="+",
+        help="Specify a list of Wilson coefficients to use in filling histograms.",
+    )
+    parser.add_argument(
+        "--hist-list",
+        action="extend",
+        nargs="+",
+        help="Specify a list of histograms to fill.",
+    )
+    parser.add_argument(
+        "--ecut",
+        default=None,
+        help="Energy cut threshold i.e. throw out events above this (GeV)",
+    )
+    parser.add_argument(
+        "--port",
+        default="9123-9130",
+        help="Specify the Work Queue port. An integer PORT or an integer range PORT_MIN-PORT_MAX.",
+    )
 
     args = parser.parse_args()
-    jsonFiles  = args.jsonFiles
```

Comment:
I personally like this older style where the `=` lined up, but it's not something I'll force. What do @Andrew42 and @sscruz think?

Reply:
I like the style where they line up too. However, I got tired of fighting black/flake8 as that's not the officially enforced style.